__tests__/project.test.js

```diff
@@ -57,3 +57,62 @@ describe("GET /api/articles/article:id", () => {
       });
   });
 });
+
+describe(" PATCH /api/articles/:article_id", () => {
+  it("status:200, responds with the updated article", () => {
+    const articleUpdates = {
+      inc_votes: 50,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(articleUpdates)
+      .expect(200)
+      .then((results) => {
+        expect(results.body).toEqual(
+          expect.objectContaining({
+            article_id: 1,
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: 150,
+          })
+        );
+      });
+  });
+  test(" 400: no inc votes on requested body", () => {
+    const articleUpdates = {};
+    return request(app)
+      .patch("/api/articles/1")
+      .send(articleUpdates)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("bad request");
+      });
+  });
+  test("invalid inc votes like a string instead of a number", () => {
+    const articleUpdates = {
+      inc_votes: "dog",
```

Comment:
I like that you've tested for both the absence of `inc_votes` and an invalid value of `inc_votes`. Nice work!

Reply:
thanks